packages/codegen.go/src/operations.ts

```diff
@@ -166,8 +173,24 @@ export async function generateOperations(codeModel: go.CodeModel): Promise<Array
       clientText += '}\n\n';
     }
 
-    // generate operations
+    // generate client accessors and operations
     let opText = '';
+    for (const clientAccessor of client.clientAccessors) {
+      const methodName = `New${clientAccessor.subClient.clientName}`;
```

Comment:
Would this work and if so, are there dragons?
```suggestion
      const methodName = `${clientAccessor.subClient.ctorName}`;
```

Reply:
Good catch, I missed `ctorName`. Fixed.